README.md

```diff
@@ -24,7 +24,8 @@ diagram](https://github.com/codeforamerica/citygram/blob/master/images/citygram_
       and details. See [Socrata Applications](https://dev.socrata.com/register) to register a new application (and
       receive a token).
 * Start dependencies
-  * `docker-compose up db`
+  * `docker-compose up -d db`
+  * `docker-compose up citygram_migration citygram_service_migration`
```

Comment:
i think you meant citygram_service**s**_migration

Reply:
🤦‍♂️ thanks!